warehouse/integrations/snowflake/snowflake_test.go

```diff
@@ -293,8 +293,6 @@ func TestIntegration(t *testing.T) {
 			tc := tc
 
 			t.Run(tc.name, func(t *testing.T) {
-				t.Parallel()
```

Comment:
This will go away anyway due to [this PR](https://github.com/rudderlabs/rudder-server/pull/3745).